english/adipose/adipose.md

```diff
@@ -4,4 +4,6 @@ You die from fat drain.
 
 You are absorbed into the marshmallow wall and now you wait with the others for new recruits.
 
-You emerge from the wall a marshmallow human. YOu now resemble the michelin man. 
\ No newline at end of file
+You emerge from the wall a marshmallow human. YOu now resemble the michelin man. 
+
+As you ponder your place in the universe, you feel a slight fire under your feet and begin to worry ... "Am I a person dreaming I'm a marshmallow being roasted alive, or am a marshmallow being roasted alive who awoke from a dream I was a human?"
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.